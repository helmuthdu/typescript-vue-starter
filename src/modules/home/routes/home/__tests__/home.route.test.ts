import { shallowMount, Wrapper } from '@vue/test-utils';
import HomeRoute from '../home.route.vue';

describe('HomeRoute component', () => {
  let wrapper: Wrapper<HomeRoute>;

  beforeEach(() => {
    wrapper = shallowMount(HomeRoute);
  });

  it('should renders with props', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
