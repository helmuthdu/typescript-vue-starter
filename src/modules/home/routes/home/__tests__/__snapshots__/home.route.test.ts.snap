// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeRoute component should renders with props 1`] = `
<v-layout-stub tag="div" column="" justify-center="" align-center="">
  <v-flex-stub tag="div" xs12="" sm8="" md6="">
    <v-card-stub tag="div">
      <v-card-title-stub class="headline">Welcome to the Vuetify + Vue.js template</v-card-title-stub>
      <v-card-text-stub>
        <p>
          Vuetify is a progressive Material Design component framework for Vue.js. It was designed to empower
          developers to create amazing applications.
        </p>
        <p>
          For more information on Vuetify, check out the
          <a href="https://vuetifyjs.com" target="_blank">documentation</a>.
        </p>
        <p>
          If you have questions, please join the official
          <a href="https://chat.vuetifyjs.com/" target="_blank" title="chat">discord</a>.
        </p>
        <p>
          Find a bug? Report it on the github
          <a href="https://github.com/vuetifyjs/vuetify/issues" target="_blank" title="contribute">issue board</a>.
        </p>
        <p>
          Thank you for developing with Vuetify and I look forward to bringing more exciting features in the future.
        </p>
        <div class="text-xs-right"><em><small>— John Leider</small></em></div>
        <hr class="my-3"> <a href="https://vuejs.org/" target="_blank">vue Documentation</a> <br> <a href="https://github.com/vue/vue.js" target="_blank">vue GitHub</a>§
      </v-card-text-stub>
      <v-card-actions-stub>
        <v-spacer-stub></v-spacer-stub>
        <v-btn-stub color="primary" activeclass="v-btn--active" to="/about" tag="button" flat="true" type="button" router="">Continue</v-btn-stub>
      </v-card-actions-stub>
    </v-card-stub>
  </v-flex-stub>
</v-layout-stub>
`;
